src/endpoints/createLanguagesEndpoint.js

```diff
@@ -10,19 +10,29 @@ export const LANGUAGES_ENDPOINT_NAME = 'languages'
 
 type ParamsType = { city: ?string }
 
-export default (baseUrl: string): Endpoint<ParamsType, Array<LanguageModel>> =>
+export default (baseUrl: string, isTunews: ?boolean): Endpoint<ParamsType, Array<LanguageModel>> =>
   new EndpointBuilder(LANGUAGES_ENDPOINT_NAME)
     .withParamsToUrlMapper(params => {
       if (!params.city) {
         throw new ParamMissingError(LANGUAGES_ENDPOINT_NAME, 'city')
       }
+      if (isTunews) {
+        return `${baseUrl}/v1/news/languages`
+      }
       return `${baseUrl}/${params.city}/de/wp-json/extensions/v3/languages`
     })
-    .withMapper((json: Array<JsonLanguageType>) => json
-      .map(language => new LanguageModel(
-        language.code,
-        language.native_name
-      ))
-      .sort((lang1, lang2) => lang1.code.localeCompare(lang2.code))
-    )
+    .withMapper((json: Array<JsonLanguageType>) => {
+      return json
+        .map(language => {
+          if ((isTunews && !language.name) || (!isTunews && !language.native_name)) {
+            throw new ParamMissingError(LANGUAGES_ENDPOINT_NAME, isTunews ? 'name' : 'native_name')
```

Comment:
```suggestion
            throw new MappingError(LANGUAGES_ENDPOINT_NAME, isTunews ? 'name' : 'native_name')
```
`ParamMissingError` is/was used to indicate the request method of the endpoint was called without a necessary parameter, e.g. `baseUrl`.

Reply:
Fixed

Reply:
The error message should also be adjusted.

Reply:
It's already adjusted in the last commit

Reply:
In this commit? https://github.com/Integreat/integreat-api-client/pull/35/commits/b781646544330ea6d848804e978c82ca48fc2c25
AFAICS it's still `isTunews ? 'name' : 'native_name'`

Reply:
Do you mean writing a more descriptive message? What do you suggest?
```js
throw new MappingError(LANGUAGES_ENDPOINT_NAME, `${isTunews ? 'name' : 'native_name'} is missing`)
```

Reply:
Yes, sorry if I wasn't clear. I'd propose `Unexpected json format. Response did not contain ${isTunews ? 'name' : 'native_name'}` or something like that.

Reply:
Looking at it again it should be clear. my bad. fixed now : )